index.html

```diff
@@ -171,10 +171,29 @@ <h5>Skills</h5>
             <li class="aboutlist">Terminal</li>
         </ul>
     </section>
-
-
-
-
+    <section class="myForm">
+        <p> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+        <form class="myForm" action="https://formspree.io/f/xwkyeara" method="post">
+            <input name="username" type="text" id="name" placeholder="Full name" maxlength="30" required>
+            <input name="useremail" type="email" id="email" placeholder="Email address" required>
+            <textarea name="usermessage" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ ] [OPTIONAL] You have done a good job with the form :muscle: but I think it could be a good idea to use the `placeholder` attribute for the "Enter text here" text for a better user experience :raised_hands: